recipes/exiv2/all/patches/0002-fpic.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index c52b352f..92fcd847 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -142,11 +142,6 @@ if (MSVC)
+     set_target_properties(exiv2lib PROPERTIES LINK_FLAGS "/ignore:4099")
+ endif()
+ 
+-set_target_properties( exiv2lib_int PROPERTIES
+-    POSITION_INDEPENDENT_CODE ON
+-    COMPILE_DEFINITIONS exiv2lib_EXPORTS
+-)
+-
+ target_include_directories(exiv2lib_int PRIVATE ${ZLIB_INCLUDE_DIR})
+ target_include_directories(exiv2lib PRIVATE 
+     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/xmpsdk/include>
+diff --git a/xmpsdk/CMakeLists.txt b/xmpsdk/CMakeLists.txt
+index a22698fb..5de35526 100644
+--- a/xmpsdk/CMakeLists.txt
++++ b/xmpsdk/CMakeLists.txt
+@@ -45,10 +45,6 @@ if (MSVC)
+     target_compile_definitions(exiv2-xmp PRIVATE XML_STATIC)
+ endif()
+ 
+-if (BUILD_SHARED_LIBS)
+-    set_property(TARGET exiv2-xmp PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
I think this should be kept.
When building shared libraries, object files must be created with `-fPIC`.

Reply:
I have provided `CMAKE_POSITION_INDEPENDENT_CODE` in the conanfile explicitly for all targets, check a comment.

Reply:
Indeed.
But for having a small patch, this part can be removed from the patch.